core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -181,6 +181,12 @@ private[spark] class EventLoggingListener(
     logEvent(event, flushLogger = true)
   override def onExecutorRemoved(event: SparkListenerExecutorRemoved) =
     logEvent(event, flushLogger = true)
+  override def onBlockUpdate(event: SparkListenerBlockUpdate) = {
+    // we only log Broadcast block update for now
```

Comment:
can you add to this comment that we don't need to log RDD blocks, b/c they are already logged as part of `StageCompleted` events